components/BottomNavigationActionLink.tsx

```diff
@@ -0,0 +1,16 @@
+import { BottomNavigationActionProps, Button, LinkProps } from "@material-ui/core"
+import React from "react"
+import Link from "next/link"
+
+export type BottomNavigationActionLinkProps = Omit<BottomNavigationActionProps, 'href' | 'classes'> &
```

Comment:
I had to use this workaround for the material-ui BottomNavigationAction and nextjs Link component interaction as BottomNavigationAction labels weren't appearing using more conventional methods. The types seem to be a bit off here.

Reply:
As long as it works I'm happy!